### ISSUE TITLE
Sort by control on search result page should be reachable with keyboard focus

### ISSUE THREAD

**Issue by sigurdm** [dart-lang/pub-dev#7243]
And the drop-down should be usable with keyboard-only interactions.

**Comment by isoos** [comment]
Comments from #7254:

> It does not quite have the behaviour I would expect. Reaching the dropdown, I would want to navigate with arrows, not continuing with tabs. Maybe it can be done with an HTML select?

> See for instance how it works with the 'All filters' drown down in Google search. (Not saying it should be the same elements but just to explain the behaviour)
